roles/docker_build_tag_push/tasks/main.yaml

```diff
@@ -117,13 +107,21 @@
 # Cleanup images to free up space
 
   - name: delete built dbforweb image
-    docker_image: name="dbforweb" state=absent
+    docker_image: 
+      name="dbforweb" 
+      state=absent
 
   - name: delete built webwtihdb image
-    docker_image: name="webwithdb" state=absent
+    docker_image: 
+      name="webwithdb" 
+      state=absent
 
   - name: delete tagged dbforweb image
-    docker_image: name="{{ ansible_docker0.ipv4.address }}:5000/dbforweb" state=absent
+    docker_image: 
+      name="{{ ansible_docker0.ipv4.address }}:5000/dbforweb" 
+      state=absent
 
   - name: delete tagged webwithdb image
-    docker_image: name="{{ ansible_docker0.ipv4.address }}:5000/webwithdb" state=absent
+    docker_image: 
+      name="{{ ansible_docker0.ipv4.address }}:5000/webwithdb" 
+      state=absent
```

Comment:
You could actually combine this into a single task and use `with_nested`.

http://docs.ansible.com/ansible/playbooks_loops.html#nested-loops